airflow/bin/cli.py

```diff
@@ -491,16 +491,19 @@ def worker(args):
     from airflow.executors.celery_executor import app as celery_app
     from celery.bin import worker
 
+    concurrency = args.concurrency or conf.get('celery', 'celeryd_concurrency')
```

Comment:
I think this will still raise an exception if arg.concurrency is not defined and neither is celeryd_concurrency in airflow.cfg